.travis.yml

```diff
@@ -27,6 +27,16 @@ after_script:
   - zeus upload -t "application/x-xunit+xml" xunit.xml
 
 matrix:
```

Comment:
There has to be a better way to include/exclude some scripts in the main `scripts` config :<

Reply:
I’m not sure how quickly Go users move to the newer versions. That’s be my only concern.

Reply:
Me neither (yet), but I'll do some research. I only know that they do a minor release exactly every 6 months.

Reply:
I asked on Go Slack channel about this.

> The Go team support version N-2, usually.

> https://golang.org/dl/ <- I'd say to go for the two latest stable versions if you need to support more than just the latest stable.

> If it's cheap to support earlier versions, I try to do so. If it's not, then I don't.

> I don’t think there has been many significant language changes since 1.0 if backwards compat is a concern

> If you need bleeding edge stuff, you can sometimes put it behind a build tag.

> Go code is generally forward compatible (as in can be compiled with a newer compiler)

> The difference might be in standard library APIs that did not exist in earlier versions.

Reply:
Right now the only issue we have is that `golint` is supporting 1.9+ only, but it can be excluded in the build.